app/services/logging/loggerConfig.spec.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+const LoggerConfig = require("./loggerConfig");
+
+describe("loggerConfig", () => {
```

Comment:
Ideally you should create mapping tests here as per https://github.com/lewster32/corporallancot/blob/master/app/services/db/dbConfig.spec.js

This is pretty boring testing, but it ensures that you do not need to write multiple tests when LoggerConfig is injected into a dependent class.